src/appkit.cs

```diff
@@ -4689,22 +4810,28 @@ interface NSCursor : NSCoding {
 		[Export ("set")]
 		void Set ();
 
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "'SetOnMouseExited' is unused and should not be called.")]
```

Comment:
The message is unclear.
First I don't think it should include the API name (it's obvious in the IDE) <- @VincentDondain that could be a check in the intro tests
Next is it really _unused_ if user code calls it ? ;-)

Maybe `"This call will be ignored by the operating system."` ? unless I misunderstood the issue :)

same applies to similar messages just below